src/main.js

```diff
@@ -244,3 +244,81 @@ let destinations = [
   ]
 
 
+  //visited and bucktelist lists
+  const visited = destinations.filter(item => item.visited === true);
+  const bucketlist = destinations.filter(item => item.visited === false);
+ 
+
+  //get list of urls
+  const getURL = (destinations) => {
+    let url= destinations.map(obj => obj.photo.valueOf());
+    return url;
+  }
+
+  //create elements for each city
+  const elements= destinations.forEach(destination => {
+
+    //picture element
+    const createImg = (url) => {
```

Comment:
One tip, about VSCode. It seems like you don't have the default VSCode Eslint/Prettier enabled. It's __very__ useful to make your code pretty as you write it (correct indentation, consistency in spacings, quotes, etc...)

Install [Prettier](https://prettier.io/) and [Eslint](https://eslint.org/) VSCode extension
![image](https://user-images.githubusercontent.com/14869087/92303053-5ed76380-ef69-11ea-902d-4b1d1dc7f679.png)
![image](https://user-images.githubusercontent.com/14869087/92303056-64cd4480-ef69-11ea-8734-2dbdcae9ec60.png)

Go to VSCode settings, tick "Format of Save" and the restart VSCode.

![image](https://user-images.githubusercontent.com/14869087/92303016-220b6c80-ef69-11ea-97e7-dc8da310a30a.png)

Reply:
@majac91 I wonder if you were able to do this? Last time I checked the code it's still not properly indented 🤔